build-apisix-runtime-debug-centos7.sh

```diff
@@ -4,16 +4,16 @@ set -x
 
 yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo
 yum -y install gcc gcc-c++ patch wget git make sudo
-yum -y install openresty-openssl111-debug-devel openresty-pcre-devel openresty-zlib-devel
+yum -y install openresty-openssl111-devel openresty-pcre-devel openresty-zlib-devel
```

Comment:
Why remove this？

Reply:
not need it anymore.
not need this path `/usr/local/openresty-debug/openssl`